### PR TITLE
Overview always has a teal color even when it's not selected #610

### PR DESCRIPTION
**Pull request by p4puniya**
## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [x] 🔧 Bug fix (non-breaking change which fixes an issue)
- [x] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [ ] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CODE_OF_CONDUCT.md) document.
- [ ] I've read the [`CONTRIBUTING.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CONTRIBUTING.md) guide.
- [ ] I've updated the code style using `make codestyle`.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

File: frontend/src/router.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default new Router({
                             },
                             children: [
                                 {
-                                    path: '',
+                                    path: 'projectOverview',
```
Comment (Googleton) [inline frontend/src/router.ts:61]: To be more consistent with other routes, maybe rename to `overview`

File: frontend/src/views/main/project/ProjectOverview.vue [hunk 2]
```diff
@@ -55,6 +55,7 @@ import Datasets from '@/views/main/project/Datasets.vue';
 import FeedbackList from '@/views/main/project/FeedbackList.vue';
 import {IUserProfileMinimal} from '@/interfaces';
 import mixpanel from "mixpanel-browser";
+import {DatasetDTO, ProjectDTO} from "@/generated-sources";
```
Comment (Googleton) [inline frontend/src/views/main/project/ProjectOverview.vue:58]: DatasetDTO is imported but I don't see where it is used

File: frontend/src/views/main/project/ProjectOverview.vue [hunk 3]
```diff
@@ -65,7 +66,7 @@ export default class ProjectOverview extends Vue {
 
   @Prop({required: true}) projectId!: number;
   @Prop({type: Boolean, required: true, default: false}) isProjectOwnerOrAdmin!: boolean;
-
+  projectOverview: ProjectDTO[] = [];
```
Comment (Googleton) [inline frontend/src/views/main/project/ProjectOverview.vue:69]: This is also unused

**Review by Googleton — CHANGES_REQUESTED** [review]
You are on the right track with the route path change, I'm not sure why changes were made to `ProjectOverview.vue` however

**Comment by p4puniya** [comment]
Seems like my recent push went as a new Pull Request. Please check it out.

**Comment by andreybavt** [comment]
hi @p4puniya , could you try pushing the [latest modification](https://github.com/Giskard-AI/giskard/pull/622/files) to the `changeOverviewColor` branch so that it appears in this PR?

**Comment by andreybavt** [comment]
☝️ failing build due to Sonar issue is a known problem for external contributors, I'll fix it soon

**Comment by p4puniya** [comment]
Done

**Review by Googleton — APPROVED** [review]
Well, looks good to me after your changes! @andreybavt I'll let you confirm and merge if it's fine for you

**Pull request merged**